core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -37,7 +39,9 @@ trait KafkaServerTestHarness extends ZooKeeperTestHarness {
   var servers: Buffer[KafkaServer] = null
   var brokerList: String = null
   var alive: Array[Boolean] = null
-
+  val kafkaPrincipalType = KafkaPrincipal.USER_TYPE
+  val clientPrincipal = s"$kafkaPrincipalType:client"
+  val kafkaPrincipal = s"$kafkaPrincipalType:kafka"
```

Comment:
Could we move clientPrincipal and kafkaPrincipal to SaslSslEndToEndAuthorizationTest since they are specific to sasl?